test-data/genops-basic.test

```diff
@@ -3058,3 +3058,198 @@ L0:
     r4 = None
     r5 = box(None, r4)
     return r5
+
+[case FinalModuleInt]
```

Comment:
Missing the leading 'test' though to be honest maybe we should just stop using that?

Reply:
I will stick to existing convention.